hedera-node/src/main/java/com/hedera/services/state/expiry/removal/AccountGC.java

```diff
@@ -64,28 +67,35 @@ public class AccountGC {
 	private final TreasuryReturnHelper treasuryReturnHelper;
 	private final BackingStore<AccountID, MerkleAccount> backingAccounts;
 	private final Supplier<MerkleMap<EntityNumPair, MerkleTokenRelStatus>> tokenRels;
+	private final Supplier<MerkleMap<EntityNumPair, MerkleUniqueToken>> uniqueTokens;
+	private final GlobalDynamicProperties dynamicProperties;
 
-	private RemovalFacilitation removalFacilitation = MapValueListUtils::inPlaceRemoveFromMapValueList;
+	private RemovalFacilitation removalFacilitation = MapValueListUtils::overwritingRemoveFromMapValueList;
```

Comment:
Why change to `overwritingRemove`? The target is a `MerkleMap`, so `getForModify()` is likely faster.